### PR TITLE
[color-name] Add type for color-name package

### PR DESCRIPTION
**Pull request by Ailrun**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/color-name/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
+// Type definitions for color-name 1.1
+// Project: https://www.npmjs.com/package/color-name
+// Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type Red = number;
```
Comment (andy-ms) [inline types/color-name/index.d.ts]: I would just do:
```ts
export type RGB = [number, number, number];

export const aliceblue: RGB;
...
...

Comment (Ailrun) [inline types/color-name/index.d.ts]: @andy-ms I fix it with `RGB`, but I think specifying `Red`, `Green`, and `Blue` is more informative.

Comment (andy-ms) [inline types/color-name/index.d.ts]: You could add a JSDoc comment to the type.

Comment (Ailrun) [inline types/color-name/index.d.ts]: @andy-ms Oh, you're right. I will fix it.

Comment (Ailrun) [inline types/color-name/index.d.ts]: @andy-ms Fixed.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**